apps/client/src/components/Atoms/DownloadButton.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
```

Comment:
I understand your thought, but Button is too general, since it needs to be hidden on `@media print`.

We can call it `PrintButton` or add a new component next to `PrintOnly`, which we call something like `BrowserOnly` or `PrintHide`?

Reply:
For now I renamed to `PrintButton`

Reply:
Since you can never interact with a button on a piece of paper I think it's a good thing to (by default) not show it in `@media print`.